app/templates/components/billing-summary-status.hbs

```diff
@@ -16,9 +16,15 @@
       {{this.subscription.status}}
     </span>
   {{else if this.subscription.isManual}}
-    <span class="badge green square" data-test-active-status='true'>
-      active
-    </span>
+    {{#if this.subscription.manualSubscriptionExpired}}
+      <span class="badge red square" data-test-expired-status='true'>
+        Expired
```

Comment:
The casing seems to be inconsistent here. I'd assume that it's handled by CSS, so it's completely optional and opinionated, but it would be nice to still have it consistent in templates (look the same as in resulting html).